models/transaction_logs.go

```diff
@@ -23,3 +31,28 @@ func TransactionLog(db *sql.DB, log TransactionLogs) error {
 		log.CreatedAt)
 	return err
 }
+
+func ListTransactionLog(db *sql.DB, id int) (list []HistoryTransaction, err error) {
+	accNumber, err := GetAccNumber(db, id)
+	if err != nil {
+		return list, err
+	}
+
+	rows, err := db.Query("SELECT account_num, dest_account, tran_amount, description, created_at FROM transaction_logs WHERE account_num = $1", accNumber)
+	if err != nil {
+		return list, err
+	}
+
+	defer rows.Close()
+	var amount float32
```

Comment:
no need to store it as float32 since sekar already updated the data type to integer